### PR TITLE
add Wangqiliang.xml for my HTTPS-ed domains.

### PR DESCRIPTION
**Pull request by wangqiliang**

### CODE REVIEW

File: src/chrome/content/Wangqiliang.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <target host="blog.qtxh.net" />
+  <target host="wp.qtxh.net" />
+  <target host="touxiang.qtxh.net" />
+  <target host="yjsw.qtxh.net" />
+  <target host="emoji.qtxh.net" />
+  <ruleset name="wangqiliang.com">
+  <target host="wangqiliang.com" />
+  <target host="cdn.wangqiliang.com" />
+  <target host="www.wangqiliang.com" />
+  <target host="chatbox.wangqiliang.com"
```
Comment (fuglede) [inline src/chrome/content/Wangqiliang.xml]: This one is missing the end of the tag.

File: src/chrome/content/Wangqiliang.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <target host="upload.qtxh.net" />
+  <target host="www.qtxh.net" />
+  <target host="g.qtxh.net" />
+  <target host="img.qtxh.net" />
+  <target host="blog.qtxh.net" />
+  <target host="wp.qtxh.net" />
+  <target host="touxiang.qtxh.net" />
+  <target host="yjsw.qtxh.net" />
+  <target host="emoji.qtxh.net" />
+  <ruleset name="wangqiliang.com">
```
Comment (fuglede) [inline src/chrome/content/Wangqiliang.xml]: This line should be removed.

**Comment by kn007** [comment]
Maybe one domain one rules file will be good for this plugin.

**Comment by kn007** [comment]
Capitalizes the first letter in the file name.

**Comment by kn007** [comment]
Oh, man, you push file to wrong path, you need to push this in `src/chrome/content/rules`, not in `src/chrome/content`.

**Comment by kn007** [comment]
Parse error: syntax error, unexpected end of file.

Two ruleset open, one ruleset end.

**Comment by kn007** [comment]
Content `*.qtxh.net` to file `Qtxh.net`
Content `*.wangqiliang.com` to file `Wangqiliang.com`
Content `*.xinbiji.cn` to file `Xinbiji.cn`
Content `*.wangql.cn` to file `Wangql.cn`
Not content `*.yjsw.sh.cn` and `*.wql.zj.cn`, or content to file `Wangqiliang.other`

**Comment by wangqiliang** [comment]
Oh Gosh!
I will correct these errors and pull it again later.

**Comment by wangqiliang** [comment]
Thanks to @kn007 @fuglede

**Pull request closed without merging**